### ISSUE TITLE
Vendor pane skin - cutting off text

### ISSUE THREAD

**Issue by LanceB71** [failcoder/Supervillain_UI#119]
The skin is cutting off the top of text in vendor windows.  Does not matter font size.  

![wowscrnshot_041215_160823](https://cloud.githubusercontent.com/assets/11777732/7108035/6dbca026-e12e-11e4-9075-1f6c9c24d3ff.jpg)

**Comment by failcoder** [comment]
Is this still happening after latest update?

**Comment by LanceB71** [comment]
Yes.  Looks like anything that is over 1 line of text (wrapped text) causes this.  Fresh screen shot.
![wowscrnshot_041715_170158](https://cloud.githubusercontent.com/assets/11777732/7212907/fc37436c-e523-11e4-8a66-f73b797fcba4.jpg)

**Comment by penguinsane** [comment]
Related, but profession trainers have a similar issue on the bottom. Money and "train" button are overlapped by other items.

This issue has been around for as long as I can remember. I reported it ages ago, but it might have gotten lost in the shuffle somewhere or another. Just remembered it when I re-read this ticket.

http://puu.sh/hpQbg/d40af63e2a.jpg

**Comment by failcoder** [comment]
This will be fixed in the next update.

**Comment by penguinsane** [comment]
Still present: 
http://puu.sh/hFyLK/e7bd651a58.jpg (text too close to edge)
http://puu.sh/hFyIp/d153e53fb6.jpg (train button under items)
